pkg/apis/k0s.k0sproject.io/v1beta1/kuberouter.go

```diff
@@ -26,6 +26,8 @@ type KubeRouter struct {
 	MetricsPort int `json:"metricsPort"`
 	// Activate Hairpin Mode (allow a Pod behind a Service to communicate to its own ClusterIP:Port)
 	HairpinMode bool `json:"hairpinMode"`
+	// IP Masquerade on the host for traffic originating from the pod network and destined outside of it (default: false)
+	IpMasq bool `json:"ipMasq"`
```

Comment:
Could I ask you to change the Go variable name to `IPMasq`. Go usually uses capital letters for all abbreviations (e.g. `HTMLAddress`, `CNIImage`, etc).